echo.coffee

```diff
@@ -97,7 +97,6 @@ module.exports = (env) =>
               /\/favicon\.ico/,
               /\/hue_logo_0\.png/,
               /\/hue_logo_3\.png/,
-              /\/api\/.+/,
```

Comment:
Did you test this change? I'm asking because i have seen requests to /api from my echo. Also other hue emulation implementations like openhab (see https://github.com/openhab/openhab2-addons/blob/master/addons/io/org.openhab.io.hueemulation/src/main/java/org/openhab/io/hueemulation/internal/HueEmulationServlet.java) expose this path for the echo